### ISSUE TITLE
Question about the unused parameters in Adapointr

### ISSUE THREAD

**Issue by pkqbajng** [yuxumin/PoinTr#129]
Hi, sir, thanks for your excellent job. However, I found that the parameters of the query ranking won't be updated during the training procedure if set find_unused_paramters=False. Could you please check its effectiveness?

**Comment by yuxumin** [comment]
Yes, it wont be optimized. The right implementation of this ranking is based on gumble-softmax, we will correct the code for this part in next push.

**Comment by pkqbajng** [comment]
Okay, thanks for your reply. When conducting ablation study on this, does you concat the generated query with the input ones and then use farthest sampling in the original experiments?
